src/vdm.c

```diff
@@ -62,14 +72,23 @@ vdm_memcpy(struct vdm *vdm, void *dest, void *src, size_t n)
 }
 
 static void
-memcpy_sync(void *runner, struct future_context *context)
+memcpy_impl(void *descriptor, struct future_context *context)
 {
 	struct vdm_memcpy_data *data = future_context_get_data(context);
 	struct vdm_memcpy_output *output = future_context_get_output(context);
 	output->dest = memcpy(data->dest, data->src, data->n);
 	data->vdm_cb(context);
 }
 
+static void
+memcpy_sync(void *descriptor, struct future_notifier *notifier,
```

Comment:
please add comments to functions

Reply:
There's not a single function comment in the framework (mea culpa, I was being lazy with my prototype code) - I can add them, but this PR is unrelated to that problem.